src/modules/tests/pass-completion/pass-completion.cat-be.reducer.ts

```diff
@@ -0,0 +1,16 @@
+
+import { PassCompletion } from '@dvsa/mes-test-schema/categories/BE';
+import { Action } from '@ngrx/store';
+
+export const initialState: PassCompletion = {
+  provisionalLicenceProvided: false,
```

Comment:
the data values need to be null else you are submitting data that may fail validation and/or isn't needed for a particular test type

Reply:
Okay, changing it

Reply:
done